### DISCUSSION TITLE
How to debug getServerSideProps on zeit?

### DISCUSSION THREAD

**Discussion by aloukissas** [vercel/vercel#4122] [General] [answered]
I seem to get a HTTP-500 on a route that works just fine locally with `now dev`. There are no logs I can use to track down the crash on zeit though :/ Any ideas?

**Comment by coetry** [comment, marked as answer]
I @aloukissas if you go to your deployment and click on the _Functions_ tabs, you'll be able to see real-time logs. If you need persistent logs, you can set up a [log drain integration](https://vercel.com/blog/log-drains).

Here's an example of [function logs](https://vercel.com/coetry/api/h75u0jq2i/functions) for a simple API I deployed.

**Reply by aloukissas** [reply]
Thanks! It might have been eventual consistency issues, because nothing showed up there in the first two attempts.
